tests/scripts/run_frequent_regressions_multi_device.sh

```diff
@@ -16,6 +16,10 @@ cd $TT_METAL_HOME
 export PYTHONPATH=$TT_METAL_HOME
 
 pytest tests/ttnn/unit_tests/test_multi_device.py
+
+pytest tests/tt_metal/microbenchmarks/ethernet/test_ethernet_bidirectional_bandwidth_microbenchmark.py -k test_bidirectional_erisc_bandwidth
```

Comment:
Hmm still looks like there's a -k here? Or are you still working on splitting it up?

Reply:
actually the -k is completely redundant now... I'll get rid of it :)